users/engler/fairseq/conversion.py

```diff
@@ -0,0 +1,216 @@
+import subprocess as sp
+import numpy as np
+import copy
+import pickle
+
+from sisyphus import Job, Task, tk
+
+from i6_core.returnn.config import ReturnnConfig
+import i6_core.util as util
+
+
+def main(
+    pytorch_config,
+    converter_kwargs,
+    fairseq_root,
+    pytorch_to_returnn_root,
+    returnn_root,
+    out_returnn_checkpoint,
+    out_returnn_model_dict,
+    model_func_name,
+):
+    if fairseq_root:
+        sys.path.append(fairseq_root)
+    if pytorch_to_returnn_root:
+        sys.path.append(pytorch_to_returnn_root)
+    if returnn_root:
+        sys.path.append(returnn_root)
+
+    from returnn.tf.util.basic import (
+        debug_register_better_repr,
+        setup_tf_thread_pools,
+        print_available_devices,
+    )
+    from returnn.log import log
+
+    better_exchook.install()
+    debug_register_better_repr()
+    log.initialize(verbosity=[5])
+    log.Verbosity = 10
+    setup_tf_thread_pools()
+    print_available_devices()
+
+    def model_func_caller(wrapped_import, inputs):
+        return eval(model_func_name)(wrapped_import, inputs, pytorch_config)
```

Comment:
Why is this needed? To allow for arbitrary names instead of `model_func`?

Reply:
yes, the input function is not limited to be named `model_func`

Reply:
```suggestion
    model_func = globals()[model_func_name]
```
This seems easier to me, what do you think?